android/expoview/build.gradle

```diff
@@ -324,7 +324,7 @@ def reactProperties = new Properties()
 def reactNative = new File("$projectDir/../../versioned-react-native")
 
 // WHEN_VERSIONING_REMOVE_FROM_HERE
-reactNative = new File("$projectDir/../../react-native-lab/react-native")
+reactNative = new File("$projectDir/..")
```

Comment:
How come it's a correct path to the react-native?

Reply:
There are two copies of `React{Android,Common}` in the `expo/expo` repo:
- `react-native-lab/react-native`, where we keep our `expo/react-native` submodule with *some* of the changes we need for React Native to be Expo-compatible
- `android`, where the directories are copied on every RN upgrade by [`ReactAndroidCodeTransformer`](https://github.com/expo/expo/blob/master/android/tools/src/main/java/host/exp/exponent/tools/ReactAndroidCodeTransformer.java). It processes React Native even further, copying it to `android` where it is used by the `android` project we use to build Expo Client.

Reply:
I think you need to run clean before build. Without this step, I believe reanimated won't rebuild.

Reply:
> run clean before build

in order to remove the `JNILibs` contents, right? If so, then CI and Expo developers when checking out the repo the first time will get an empty `JNILibs` directory (since I removed it in the other commit) so Gradle will know to build `reanimated`.

Reply:
Looking at the CI — it's building reanimated as I write (11:10:34 timestamp) — https://github.com/expo/expo/pull/10061/checks?check_run_id=1071534925

Reply:
To clear buildDir